app/javascript/components/supplemental_application/SupplementalApplicationContainer.js

```diff
@@ -197,7 +195,26 @@ class SupplementalApplicationContainer extends React.Component {
               <DemographicsSection />
               <ScrollableAnchor id={'status-history-section'}><div><StatusHistorySection /></div></ScrollableAnchor>
               <div className='padding-bottom--2x margin-bottom--2x' />
-              <ActionButtons loading={loading} />
+              <div className='button-pager'>
```

Comment:
This was `<ActionButtons/`> but I needed to access the `state`, so I included it in this main component and deleted `<ActionButtons/`> .